utils/fotos/update.py

```diff
@@ -41,24 +40,26 @@ def scan(album):
                     'name': name,
                     'random': random.random(),
                     'visibility': ['hidden']})
+                subalbum.update_metadata(album)
+                subalbum.save()
```

Comment:
Nee, `update_metadata` slaat EXIF metadata niet op, alleen `effectiveVisibility`.
Dat `effectiveVisibility` wél wordt opgeslagen is omdat het daar om een recursieve functie gaat. Ik zou dit kunnen aanpassen, aangezien dit inderdaad best lelijk is zo.